daml-assistant/integration-tests/src/DA/Daml/Assistant/IntegrationTests.hs

```diff
@@ -8,11 +8,13 @@ import Control.Concurrent.STM
 import Control.Exception.Extra
 import Control.Lens
 import Control.Monad
-import Control.Monad.Loops (untilM_)
+-- import Control.Monad.Loops (untilM_)
```

Comment:
leftover?

Reply:
no, it's from commenting out the hot reload test, until I can reenable it